.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  build-and-test:
+    name: build-and-test on ${{matrix.os}}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest]
```

Comment:
Any reason this can't be ubuntu?

Reply:
Project isn't built to handle it and doesn't seem worth the time to make it cross compatible at this point since it has one function and is used by mainly one person who uses a windows machine

(https://github.com/IntelliTect/ListingManager/actions/runs/3177500092/jobs/5178010334)